kitsune/settings.py

```diff
@@ -881,6 +884,9 @@ def JINJA_CONFIG():
 # Products
 PRODUCT_IMAGE_PATH = "uploads/products/"
 
+# Banners
+BANNER_IMAGE_PATH = "uploads/banners/"
```

Comment:
Can't we use the product path?

Reply:
I think it ought to be different, like the topic image path is.